images/linux/Ubuntu2004-Readme.md

```diff
@@ -128,41 +131,41 @@
 | CE 22.1.0 | GRAALVM_11_ROOT       |
 
 ### PHP
-| Tool     | Version             |
-| -------- | ------------------- |
-| PHP      | 7.4.29 8.0.18 8.1.5 |
-| Composer | 2.3.5               |
-| PHPUnit  | 8.5.26              |
+| Tool     | Version       |
+| -------- | ------------- |
+| PHP      | 7.4.29  8.1.6 |
```

Comment:
8.0.18 - is unexpectly missing

Reply:
@ddobranic, let's update php version manually and fix `Get-PHPVersions` function.

Reply:
OK.

Reply:
added 8.0.19

Reply:
issue https://github.com/actions/virtual-environments-internal/issues/3806